### PR TITLE
Windows Spheres of Support for Dedicated and Managed Operations

### PR DESCRIPTION
**Pull request by b1naryh3r0**
New high level Rackspace SoS reviewed by WindowsTC

### CODE REVIEW

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-63da15bae2/how-to/)

File: content/dedicated-hosting/windows-spheres-of-support-for-dedicated-and-managed-ops.md [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+last_modified_by: JP Gonzalez
+product: Dedicated Hosting
+product_url: dedicated-hosting
+---
+
+This article outlines our **Fanatical Support** of software and server configurations for machines (virtual and physical) running a supported Windows operating system and represents a high level overview for offerings found in Rackspace. Please contact Rackspace sales or your account representative for more information regarding the components of specific Windows Roles and Features supported in Rackspace offerings.
+
+
+
+### Operating Systems
```
[the remark this answers is not in the document]
Comment (b1naryh3r0) [reply content/dedicated-hosting/windows-spheres-of-support-for-dedicated-and-managed-ops.md]: Used in the context of OS the heading is Operating System representing the acronym.  Thank you for the suggestion.

File: content/dedicated-hosting/windows-spheres-of-support-for-dedicated-and-managed-ops.md [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+type: article
+created_date: '2018-08-02'
+created_by: JP gonzalez
+last_modified_date: '2018-08-02'
+last_modified_by: JP Gonzalez
+product: Dedicated Hosting
+product_url: dedicated-hosting
+---
+
+This article outlines our **Fanatical Support** of software and server configurations for machines (virtual and physical) running a supported Windows operating system and represents a high level overview for offerings found in Rackspace. Please contact Rackspace sales or your account representative for more information regarding the components of specific Windows Roles and Features supported in Rackspace offerings.
```
[the remark this answers is not in the document]
Comment (b1naryh3r0) [reply content/dedicated-hosting/windows-spheres-of-support-for-dedicated-and-managed-ops.md]: Changed

File: content/dedicated-hosting/windows-spheres-of-support-for-dedicated-and-managed-ops.md [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+| Windows Server 2008    | Yes       |                         |                        |                    |                   |                      |
+| Windows Server 2008 R2 | Yes       |                         |                        | Yes                |                   |                      |
+| Windows Server 2012    | Yes       |                         |                        |                    |                   |                      |
+| Windows Server 2012 R2 | Yes       | Yes                     | Yes                    | Yes                | Yes               | Yes                  |
+| Windows Server 2016    | Yes       | Yes                     | Yes                    | Yes                | Yes               | Yes                  |
+
+*Note**: Rackspace makes every effort to align our support dates for operating systems with the manufacturer's support dates. If a manufacturer decides to shorten the support life of an operating system, Rackspace might be forced to end support sooner than originally anticipated. For full details on support life, see the [Rackspace EOL Terms](https://www.rackspace.com/information/legal/eolterms).
```
[the remark this answers is not in the document]
Comment (b1naryh3r0) [reply content/dedicated-hosting/windows-spheres-of-support-for-dedicated-and-managed-ops.md]: Updated syntax

File: content/dedicated-hosting/windows-spheres-of-support-for-dedicated-and-managed-ops.md [hunk 4]
```diff
@@ -0,0 +1,182 @@
...
+| FTPS           | Yes          | Yes           | Yes             |
+| SMTP           | Yes          | Yes           | Yes             |
+| PHP            | Yes          | Yes           | Yes             |
+| PHP Extensions | Yes          | Yes           | Yes             |
+
+**Note**: Not all add-on modules are supported. Contact Rackspace Support for more information.
+
+
+
+### Active Directory
```
[the remark this answers is not in the document]
Comment (b1naryh3r0) [reply content/dedicated-hosting/windows-spheres-of-support-for-dedicated-and-managed-ops.md]: Used in the context of AD the heading is Active Directory representing the acronym. Thank you for the suggestion.

File: content/dedicated-hosting/windows-spheres-of-support-for-dedicated-and-managed-ops.md [hunk 5]
```diff
@@ -0,0 +1,182 @@
...
+
+| Feature / Operation | Installation | Configuration | Troubleshooting |
```
[the remark this answers is not in the document]
Comment (b1naryh3r0) [reply content/dedicated-hosting/windows-spheres-of-support-for-dedicated-and-managed-ops.md]: Updated

File: content/dedicated-hosting/windows-spheres-of-support-for-dedicated-and-managed-ops.md [hunk 6]
```diff
@@ -0,0 +1,182 @@
...
+| MSSQL 2008    | Yes                 | Yes                 |                     |                        |              |
+| MSSQL 2008 R2 | Yes                 | Yes                 |                     |                        |              |
+| MSSQL 2012    |                     |                     | Yes                 | Yes                    |              |
+| MSSQL 2014    |                     |                     | Yes                 | Yes                    | Yes          |
+| MSSQL 2016    |                     |                     |                     | Yes                    | Yes          |
+
+
+## Supported Platforms for Microsoft SQL Editions
+
+| SQL Edition | Licensing | Installation | Troubleshooting | Managed Backup |
```
[the remark this answers is not in the document]
Comment (b1naryh3r0) [reply content/dedicated-hosting/windows-spheres-of-support-for-dedicated-and-managed-ops.md]: updated

File: content/dedicated-hosting/windows-spheres-of-support-for-dedicated-and-managed-ops.md [hunk 7]
```diff
@@ -0,0 +1,182 @@
...
+|-------------|-----------|--------------|-----------------|----------------|
+| Web         | Yes       | Yes          | Yes             | Yes            |
+| Standard    | Yes       | Yes          | Yes             | Yes            |
+| Enterprise  | Yes       | Yes          | Yes             | Yes            |
+
+**Note**: Customers using an MSDN license for Microsoft SQL are free to install and run this edition of SQL, but Rackspace cannot provide the license, install the software, monitor it, or back it up using Rackspace managed Backup SQL agent. Also note, the use production data in an instance licensed via MSDN is strictly prohibited.
```
[the remark this answers is not in the document]
Comment (b1naryh3r0) [reply content/dedicated-hosting/windows-spheres-of-support-for-dedicated-and-managed-ops.md]: Updated:
**Note**: Customers using an MSDN license for Microsoft SQL are free to install and run this edition of SQL. But Rackspace cannot provide a MSDN license, install MSDN software, monitor MSDN applications, or back up the MSDN applications up using Rackspace Managed Backup for MSSQL offering. Also note, the use production data in an instance licensed via MSDN is strictly prohibited.

File: content/dedicated-hosting/windows-spheres-of-support-for-dedicated-and-managed-ops.md [hunk 8]
```diff
@@ -0,0 +1,182 @@
...
+| SQL Edition | Licensing | Installation | Troubleshooting | Managed Backup |
+|-------------|-----------|--------------|-----------------|----------------|
+| Web         | Yes       | Yes          | Yes             | Yes            |
+| Standard    | Yes       | Yes          | Yes             | Yes            |
+| Enterprise  | Yes       | Yes          | Yes             | Yes            |
+
+**Note**: Customers using an MSDN license for Microsoft SQL are free to install and run this edition of SQL, but Rackspace cannot provide the license, install the software, monitor it, or back it up using Rackspace managed Backup SQL agent. Also note, the use production data in an instance licensed via MSDN is strictly prohibited.				
+
+
+## Supported Platforms for Microsoft SQL Licenses
```
[the remark this answers is not in the document]
Comment (b1naryh3r0) [reply content/dedicated-hosting/windows-spheres-of-support-for-dedicated-and-managed-ops.md]: updated

File: content/dedicated-hosting/windows-spheres-of-support-for-dedicated-and-managed-ops.md [hunk 9]
```diff
@@ -0,0 +1,182 @@
...
+| Customer provided licenses   | Yes          | Yes     | Yes       |                            | Yes             | Yes           | Yes               | Yes               |
+| AWS image library licenses   |              |         |           |                            | Yes             | Yes           |                   |                   |
+| AWS RDS                      |              |         |           |                            | Yes             | Yes           |                   |                   |
+| Azure image library licenses |              |         |           |                            |                 |               | Yes               | Yes               |
+| Azure SQL Database           |              |         |           |                            |                 |               | Yes               | Yes               |
+
+**Note**: Licensing limited to bare metal servers and Enterprise edition on hypervisors only.
+
+
+## Supported Platforms for Microsoft SQL Installation
```
[the remark this answers is not in the document]
Comment (b1naryh3r0) [reply content/dedicated-hosting/windows-spheres-of-support-for-dedicated-and-managed-ops.md]: updated

File: content/dedicated-hosting/windows-spheres-of-support-for-dedicated-and-managed-ops.md [hunk 10]
```diff
@@ -0,0 +1,182 @@
...
+| Rackspace owned licenses     | Yes*         | Yes     | Yes       | Yes                        |                 |               |                   |                   |
+| Customer provided licenses   | Yes          | Yes     | Yes       |                            | Yes             | Yes           | Yes               | Yes               |
+| AWS image library licenses   |              |         |           |                            | Yes             | Yes           |                   |                   |
+| AWS RDS                      |              |         |           |                            | Yes             | Yes           |                   |                   |
+| Azure image library licenses |              |         |           |                            |                 |               | Yes               | Yes               |
+| Azure SQL Database           |              |         |           |                            |                 |               | Yes               | Yes               |
+
+**Note**: Licensing limited to bare metal servers and Enterprise edition on hypervisors only.
```
[the remark this answers is not in the document]
Comment (b1naryh3r0) [reply content/dedicated-hosting/windows-spheres-of-support-for-dedicated-and-managed-ops.md]: updated

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-65c7ba751f/how-to/). :bow:

File: content/dedicated-hosting/windows-spheres-of-support-for-dedicated-and-managed-ops.md [hunk 11]
```diff
@@ -0,0 +1,182 @@
...
+### Active Directory
+
+| Feature / Operation | Installation | Configuration | Troubleshooting |
+|---------------------|--------------|---------------|-----------------|
+| Domain Services     | Yes          | Yes           | Yes             |
+| DNS                 | Yes          | Yes           | Yes             |
+
+
+
+### Remote Desktop Services
```
[the remark this answers is not in the document]
Comment (b1naryh3r0) [reply content/dedicated-hosting/windows-spheres-of-support-for-dedicated-and-managed-ops.md]: Used in the context of RDS the heading is Remote Desktop Services representing the acronym. Thank you for the suggestion.

File: content/dedicated-hosting/windows-spheres-of-support-for-dedicated-and-managed-ops.md [hunk 12]
```diff
@@ -0,0 +1,182 @@
...
+
+| Feature / Operation             | Installation | Configuration | Troubleshooting |
```
[the remark this answers is not in the document]
Comment (b1naryh3r0) [reply content/dedicated-hosting/windows-spheres-of-support-for-dedicated-and-managed-ops.md]: Updated

File: content/dedicated-hosting/windows-spheres-of-support-for-dedicated-and-managed-ops.md [hunk 13]
```diff
@@ -0,0 +1,182 @@
...
+|---------------------------------|--------------|---------------|-----------------|
+| Remote Desktop Licensing Server | Yes          | Yes           | Yes             |
+| Remote Desktop Session Host     | Yes          | Yes           | Yes             |
+
+
+
+### Miscellaneous Roles and Features
```
[the remark this answers is not in the document]
Comment (b1naryh3r0) [reply content/dedicated-hosting/windows-spheres-of-support-for-dedicated-and-managed-ops.md]: Updated

File: content/dedicated-hosting/windows-spheres-of-support-for-dedicated-and-managed-ops.md [hunk 14]
```diff
@@ -0,0 +1,182 @@
...
+| ODBC                            | Yes          | Yes           | Yes             |
+| Failover Clustering             | Yes          | Yes           | Yes             |
+| File and Storage Services       | Yes          | Yes           | Yes             |
+| Windows Firewall                | Yes          |               |                 |
+| Windows Server Backup           | Yes          |               |                 |
+| DFS-R (Distributed File System) | Yes          | Yes           | Yes             |
+
+
+
+### Additional Products and Services
```
[the remark this answers is not in the document]
Comment (b1naryh3r0) [reply content/dedicated-hosting/windows-spheres-of-support-for-dedicated-and-managed-ops.md]: Updated

File: content/dedicated-hosting/windows-spheres-of-support-for-dedicated-and-managed-ops.md [hunk 15]
```diff
@@ -0,0 +1,182 @@
...
+
+**Note**: Additional Rackspace licensing and service agreements required.
+
+
+
+### Microsoft SQL
+
+The following types of database servers are supported in the indicated areas:
+
+## Rackspace Supported OS and MSSQL Combinations
```
[the remark this answers is not in the document]
Comment (b1naryh3r0) [reply content/dedicated-hosting/windows-spheres-of-support-for-dedicated-and-managed-ops.md]: Updated

File: content/dedicated-hosting/windows-spheres-of-support-for-dedicated-and-managed-ops.md [hunk 16]
```diff
@@ -0,0 +1,182 @@
...
+
+| SQL Version   | Windows 2008 32 bit | Windows 2008 64 bit | Windows 2012 64 bit | Windows 2012 R2 64 bit | Windows 2016 |
```
[the remark this answers is not in the document]
Comment (b1naryh3r0) [reply content/dedicated-hosting/windows-spheres-of-support-for-dedicated-and-managed-ops.md]: Updated

File: content/dedicated-hosting/windows-spheres-of-support-for-dedicated-and-managed-ops.md [hunk 17]
```diff
@@ -0,0 +1,182 @@
...
+|---------------|---------------------|---------------------|---------------------|------------------------|--------------|
+| MSSQL 2008    | Yes                 | Yes                 |                     |                        |              |
+| MSSQL 2008 R2 | Yes                 | Yes                 |                     |                        |              |
+| MSSQL 2012    |                     |                     | Yes                 | Yes                    |              |
+| MSSQL 2014    |                     |                     | Yes                 | Yes                    | Yes          |
+| MSSQL 2016    |                     |                     |                     | Yes                    | Yes          |
+
+
+## Supported Platforms for Microsoft SQL Editions
```
[the remark this answers is not in the document]
Comment (b1naryh3r0) [reply content/dedicated-hosting/windows-spheres-of-support-for-dedicated-and-managed-ops.md]: Updated

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-90775c70f4/how-to/)

File: content/dedicated-hosting/windows-spheres-of-support-for-dedicated-and-managed-ops.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (RackerKate) [reply content/dedicated-hosting/windows-spheres-of-support-for-dedicated-and-managed-ops.md]: Should this be something like "...the use of production data?"

[the remark this answers is not in the document]
Comment (b1naryh3r0) [reply content/dedicated-hosting/windows-spheres-of-support-for-dedicated-and-managed-ops.md]: Agreed, please proceed with the change.

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-d0a4622e09/how-to/). :bow:

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-55dc699334/how-to/)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-dd10d9936a/how-to/). :bow:

**Pull request merged**